### PR TITLE
troubleshoot: update locations for daemon logfiles

### PR DESCRIPTION
**Pull request by thaJeztah**

### CODE REVIEW

File: config/daemon/index.md [hunk 1]
```diff
@@ -220,15 +220,28 @@ The daemon logs may help you diagnose problems. The logs may be saved in one of
...
-| Windows               | `AppData\Local`                                                                          |
+| Operating system          | Location                                                                                                                                 |
+|:--------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------|
+| Linux                     | Use the command `journalctl -xu docker.service` (or read `/var/log/syslog` or `/var/log/messages`, depending on your Linux Distribution) |
+| macOS (`dockerd` logs)    | `~/Library/Containers/com.docker.docker/Data/log/vm/dockerd.log`                                                                         |
+| macOS (`containerd` logs) | `~/Library/Containers/com.docker.docker/Data/log/vm/containerd.log`                                                                      |
+| Windows                   | `AppData\Local`                                                                                                                          |
```
Comment (thaJeztah) [inline config/daemon/index.md:228]: Wasn't sure if this location is still correct; probably needs two separate mentions (WSL2 + Native Windows containers)

@djs55 @StefanScherer

Comment (StefanScherer) [reply config/daemon/index.md:228]: @thaJeztah for WSL 2 it is `AppData\Roaming\Docker\log\vm\dockerd.log` and `AppData\Roaming\Docker\log\vm\containerd.log`
Windows container mode uses the Event Log AFAIK.

Comment (thaJeztah) [reply config/daemon/index.md:228]: Thanks; I'll add those

**Comment by thaJeztah** [comment]
@StefanScherer updated; ptal

**Review by StefanScherer — APPROVED** [review]
LGTM

**Pull request merged**